inventory.py

```diff
@@ -227,8 +328,9 @@ def run_course_inventory() -> None:
     enrollment_df.to_sql('enrollment', db_creator_obj.engine, if_exists='append', index=False)
     logger.info(f'Inserted data into enrollment table in {db_creator_obj.db_name}')
 
-    delta = datetime.now() - start
-    logger.info(f'Duration of run: {delta.total_seconds()}')
+    delta = time.time() - start
+    str_time = time.strftime("%H:%M:%S", time.gmtime(delta))
+    logger.info(f'Duration of run: {str_time}')
```

Comment:
This is nicely printing the time taken for all the call like this `INFO:__main__:Duration of run: 01:06:46`

Reply:
Is there a reason we can't use `datetime` for this?

Reply:
I wanted in a format of H:M:S so that based on the time take it to adjust itself to seconds, minutes etc. I did not see that with existing approach was doing it so I changed it. I don't have a specific reason for why I chose this

Reply:
Not sure the best way to do this. It looks like you're passing seconds into `gmtime`, which is really calculating the time since the epoch began, and then you're just looking at the hours, minutes, and seconds. That's probably fine? I'm not married to my approach either; the `timedelta` objects seemed appropriate here, but you'd have to do a little math to get the hours and minutes. If you decided to stick to your approach, please remove the `datetime` import.

Reply:
Done